Library/Formula/dfc.rb

```diff
@@ -1,15 +1,23 @@
-require 'formula'
-
 class Dfc < Formula
-  homepage 'http://projects.gw-computing.net/projects/dfc'
-  url 'http://projects.gw-computing.net/attachments/download/79/dfc-3.0.4.tar.gz'
-  sha1 'e3b7fc7474f2ca36c1370a3dcbcd3d1020766f1a'
+  homepage "http://projects.gw-computing.net/projects/dfc"
+  url "http://projects.gw-computing.net/attachments/download/467/dfc-3.0.5.tar.gz"
+  sha256 "3c947a1d6bc53347b1643921dcbf4c6f8fe7eb6167fc1f4e9436366f036d857a"
+
+  head "https://github.com/Rolinh/dfc.git"
 
-  depends_on 'cmake' => :build
-  depends_on 'gettext'
+  depends_on "cmake" => :build
+  depends_on "gettext"
 
   def install
-    system "cmake", ".", *std_cmake_args
-    system "make install"
+    mkdir "build" do
+      system "cmake", "..", *std_cmake_args
+      system "make", "install"
+    end
+  end
+
+  test do
+    system bin/"dfc", "-T"
+    `#{bin}/dfc -e csv > test.csv`
```

Comment:
Pretty sure this will create the file regardless. Better to use `shell_output` and check for a non-empty string without the `>`